front/app/components/UI/QuillEditor/index.tsx

```diff
@@ -852,4 +853,13 @@ const QuillEditor = memo<Props & InjectedIntlProps>(
   }
 );
 
-export default injectIntl(QuillEditor);
+const RtlQuil = styled(QuillEditor)`
```

Comment:
Instead of creating a new component here, when `isRtl` let's add the styles to the existing other `.ql-editor` styles in the `Container` styled component above.